3529/1/gemhandler.rb

```diff
@@ -0,0 +1,108 @@
+require 'nokogiri'
+require 'json'
+require 'net/http'
+require 'uri'
+require 'open-uri'
+require 'httparty'
+require 'open_uri_redirections'
+require 'yaml'
+require 'octokit'
+require 'typhoeus'
+
+class GemHendler
+  attr_reader :data_about_gem
+
+  def initialize(github_url, gem_name)
+    @url = github_url
+    @data_about_gem = {}
+    puts "Didn't find repository on github" unless @url
+    @repo_addr = adress_handle
+    @client = Octokit::Client.new(login: 'gannagoodkevich', password: 'Pusivill1999')
```

Comment:
Move credentials to separate yml file

Reply:
I don't get it:(
Could you explain more what do you mean, please?

Reply:
In real project nobody put logins and passwords, tokens, keys in code. One of approaches to put them in yml file. And you can easily read them(passwords and etc.). All popular libraries (example Rails) use inside libraries for reading yml. In your case you can use `YAML` library

Reply:
Thank you, I solved it:)

Reply:
I changed everything. It is commit fix initialize